ci/taos/config/config-plugins-format.sh

```diff
@@ -150,3 +150,10 @@ source ${REFERENCE_REPOSITORY}/ci/taos/plugins-good/${format_plugins[idx]}.sh
 echo "[MODULE] plugins-staging: Plugin group that does not have an evaluation and aging test enough"
 # Please append your plugin modules here.
 
+format_plugins[++idx]="pr-format-signed-off-by"
+echo "${format_plugins[idx]} is starting."
+echo "[MODULE] TAOS/${format_plugins[idx]}: Check 'Signed-off-by' in commit body"
+echo "[DEBUG] The current path: $(pwd)."
+echo "[DEBUG] source ${REFERENCE_REPOSITORY}/ci/taos/plugins-staging/${format_plugins[idx]}.sh"
```

Comment:
Let's put the module in plugins-good folder. 
**s/plugins-staging/plugins-good/**